app/services/analytics_events.rb

```diff
@@ -594,6 +594,59 @@ def idv_forgot_password_confirmed
     track_event('IdV: forgot password confirmed')
   end
 
+  # GPO address letter requested
+  def idv_gpo_address_letter_requested
+    track_event('IdV: USPS address letter requested')
+  end
+
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # GPO address submitted
+  def idv_gpo_address_submitted(
+    success:,
+    errors:,
+    **extra
+  )
+    track_event(
+      'IdV: USPS address submitted',
+      success: success,
+      errors: errors,
+      **extra,
+    )
+  end
+
+  # @param [Boolean] letter_already_sent
+  # GPO address visited
+  def idv_gpo_address_visited(
+    letter_already_sent:,
+    **extra
+  )
+  track_event(
+    'IdV: USPS address visited',
+    letter_already_sent: letter_already_sent,
+    **extra,
+  )
+  end
+
+  # @param [Boolean] success
+  # @param [Hash] errors
+  # @param [Hash] pii_like_keypaths
+  # GPO verification submitted
+  def idv_gpo_verification_submitted(
```

Comment:
This event had a `# Previously: ` note...
```
  IDV_GPO_VERIFICATION_SUBMITTED = 'IdV: GPO verification submitted' # Previously: "Account verification submitted"
```

 can we annotate that as well?
```suggestion
  # @identity.idp.previous_event_name Account verification submitted
  # @param [Boolean] success
  # @param [Hash] errors
  # @param [Hash] pii_like_keypaths
  # GPO verification submitted
  def idv_gpo_verification_submitted(
```

Reply:
Added to the annotation.

Reply:
Please take a look at my suggested comment, we have a specific YARD tag for this, `@identity.previous_event_name`

Reply:
Oh I see it now. Suggested comment added.